packages/react/src/components/SignInButton.tsx

```diff
@@ -11,21 +12,27 @@ export const SignInButton = withClerk(({ clerk, children, ...props }: WithClerkP
   const child = assertSingleChild(children)('SignInButton');
 
   const clickHandler = () => {
-    const opts = {
+    const opts: SignInProps = {
+      forceRedirectUrl,
+      fallbackRedirectUrl,
```

Comment:
The modal mode relies on the non-prefixed props, adding the type here revealed the issue.